ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -189,7 +189,20 @@ StackLayout {
             Layout.preferredHeight: height
             color: "transparent"
             
+            Connections {
+                target: chatsModel
+                onLoadingMessagesChanged:
+                    if(value){
+                        loadingMessagesIndicator.active = true
+                    } else {
+                         timer.setTimeout(function(){ 
+                            loadingMessagesIndicator.active = false;
+                        }, 5000);
```

Comment:
I guess this timeout is just to take into account the fact that the mailserver returns a signal before the fetch actually finishes?

Reply:
yes. It's to simulate that it's fetching messages, because otherwise, the loading indicator would dissapear too quickly since the mailserver signal is received almost immediatly